<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample RoboDK API Implementation in C++</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLoadFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowRoboDK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show RoboDK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHideRoboDK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide RoboDK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTestButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectRobot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Robot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetPosition">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve
Current
Position</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>296</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Program Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtProgName">
       <property name="text">
        <string>TestProgram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnProgRun">
       <property name="text">
        <string>Run Program</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>185</width>
      <height>114</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Joint Values</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMoveJoints">
           <property name="text">
            <string>Move to Joints</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="txtJoints"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Cartesian Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMovePose">
           <property name="text">
            <string>Move to Position</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="txtXYZWPR"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>158</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnTXn">
       <property name="text">
        <string>Tx-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnTXp">
       <property name="text">
        <string>Tx+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnTYn">
       <property name="text">
        <string>Ty-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnTYp">
       <property name="text">
        <string>Ty+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnTZn">
       <property name="text">
        <string>Tz-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnTZp">
       <property name="text">
        <string>Tz+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnRXn">
       <property name="text">
        <string>Rx-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnRXp">
       <property name="text">
        <string>Rx+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnRYn">
       <property name="text">
        <string>Ry-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnRYp">
       <property name="text">
        <string>Ry+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="btnRZn">
       <property name="text">
        <string>Rz-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btnRZp">
       <property name="text">
        <string>Rz+</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Step (mm or deg)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spnStep">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
